.github/workflows/e2e-faucet-balance.yml

```diff
@@ -17,12 +17,21 @@ jobs:
     name: balance-and-fund
     runs-on: ubuntu-latest
     steps:
+      - uses: google-github-actions/auth@v2
+        if: ${{ env.SECRETS_AVAILABLE }}
```

Comment:
should this env be set in the first place, like how it's done in the above workflow? Otherwise I assume this will always evaluate to false?

Reply:
Fixed in 0d7a8610c3516f89515855ef058f3f6006ea8c25, d40b7c187383313bcc626e4dcc394de3c94f79bb and 5ad04929165f70ba61cfd944971fcf065a28f65d; example passing run: https://github.com/valora-inc/wallet/actions/runs/12039132275/job/33566304674.